README_CONTAINER_IMAGE.md

```diff
@@ -38,4 +44,20 @@ Here is an example of how to run a containerized `openshift-ansible` playbook th
            -e PLAYBOOK_FILE=playbooks/certificate_expiry/default.yaml \
            openshift/openshift-ansible
 
-The [playbook2image examples](https://github.com/aweiteka/playbook2image/tree/master/examples) provide additional information on how to use an image built from it like this one.
+#### Scheduled job
```

Comment:
Good point, will add that x-ref.

About the file extension: I used `.yaml` to match the other files in the directory... but the style guide is quite clear about using `.yml`, and if we look under `playbooks/` there's only 11 files with `.yaml` vs 150+ with `yml`.

As a step towards compliance with the style guide: any objections if I use this PR to rename all the playbooks inside `playbooks/certificate_expiry/` from `.yaml` to `.yml`? (would update the readme accordingly)